adage/analyze/models.py

```diff
@@ -210,6 +223,8 @@ class Participation(models.Model):
     """
     node = models.ForeignKey(Node, on_delete=models.PROTECT)
     gene = models.ForeignKey(Gene, on_delete=models.PROTECT)
+    participation_type = models.ForeignKey(ParticipationType,
+                                           on_delete=models.PROTECT)
 
     class Meta:
         unique_together = ('node', 'gene')
```

Comment:
Please confirm that `unique_together` should include `participation_type` or not.